examples/project_fully_featured/project_fully_featured/__init__.py

```diff
@@ -1 +1 @@
-from .repository import hacker_news_repository
+from .repository import hacker_news_repository as hacker_news_repository
```

Comment:
Do we know why it is doing this? This might confuse users in public examples

Reply:
`ruff` is not doing this-- I added the redundant aliases. Redundant aliasing is a more easily maintainable alternative to `__all__` that we already use it top level Dagster `__init__.py`-- there are some notes on it there: https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/__init__.py

Two main purposes here:

- mark the imports as public symbols for `pyright`
- prevent deletion of them as "unused" by `ruff`

Reply:
Let's keep this out of the this diff then. These examples are not really meant for public exports, and I think this will just cause confusion